Updated developer documentation for latest changes (#75)
c6o/docs: content/concepts/teleport.md

> czctl deployment teleport my-service -n my-namespace -f ./my-service.env
```

> The user can specify one of four formats for the file where the environment variables will be written with the -m flag, specifying one of four formats:

Reviewer: ```suggestion
The user can specify one of four formats for the environment variables output, using the -m flag:
```

1. sh (sourceable shell), DEFAULT
1. env (.env format file)
1. json
1. yaml

> Whilst teleport is running, the environment file will be kept in sync as environment variables are updated and changed in the cluster.

Reviewer: ```suggestion

Whilst teleport is running, the environment file will be kept in sync as environment variables are updated and changed in the cluster.
```
